server/game/player.js

```diff
@@ -792,7 +792,10 @@ class Player extends Spectator {
             return;
         }
 
-        if(!character.dupes.isEmpty() && allowSave) {
+        if(!character.canBeKilled()) {
+            this.game.addMessage('{0} controlled by {1} cannot be killed',
+                                 this, character);
```

Comment:
Message args are slightly out of order - this will print 'player1 controlled by Tywin Lannister cannot be killed'